apps/authentication-service/src/shared/auth/authService.test.ts

```diff
@@ -0,0 +1,58 @@
+import fs from "fs";
+import { _testing, getAccessToken } from "./authService";
```

Comment:
I'm not a fan of making and exposing functions that are only used for tests. Still reading through these, but I'm sure there's a better way.